assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -154,11 +154,11 @@ const renderView = (attributes, toAttribute) => {
           />
         </>
       }
-      {!loading && !covers.length && !isExample ?
-        <div className='EmptyMessage'>
+      {!loading && !covers.length && !isExample
+        ? <div className="EmptyMessage">
```

Comment:
We should find an `eslint` rule to validate this kind of `classnames`. 
Do you think if it's possible?